manifests/init.pp

```diff
@@ -151,6 +157,17 @@
       key     => $certs::apache::apache_key,
       ca_cert => $certs::ca_cert,
       require => Class['certs::apache'],
+      port    => $docker_registry_port,
+    }
+  }
+
+  if $pulp_master {
+    $check_file = file('/usr/sbin/foreman-rake', '/dev/null')
```

Comment:
@ekohl - I am trying to test for the existence of a file, can you suggest better? This did work as is on a devel vs. prod install.